src/services/db_project_handler.py

```diff
@@ -117,4 +126,136 @@ def get_project_privileges(db: Session, username: str):
+            query_for_same_uuid = db.execute(select(Documents).where(Documents.s3_key == str(new_s3_key)))
+        
+        new_document = Documents(name=doc_name.strip().replace(" ","-"),
+                                 project_id=project_id,
+                                 added_by=caller,
+                                 content_type=content_type,
+                                 s3_key=str(new_s3_key),
+                                 added_on=datetime.now())
+        db.add(new_document)
+        # upload to s3
+        try:
+            upload_file_to_s3(bucket_name="project-manager-documents",
+                              key=str(new_s3_key),
+                              bin_file=byfile)
+        except Exception as ex:
+            raise ex
         
+        # commit added document only if upload to s3 was successful
+        db.commit()
+        # return the created document to the user
+        final_document = db.get(Documents, new_document.id)
+        return ProjectDocument(id=final_document.id,
+                               name=final_document.name,
+                               added_by=final_document.added_by,
+                               added_on=final_document.added_on,
+                               content_type=final_document.content_type,
+                               project_id=final_document.project_id)
+
+
+    def get_docs(self,
+                 project_id: int,
+                 db: Session):
+        all_documents = db.execute(select(Documents.id,
+                                          Documents.name,
+                                          Documents.added_by,
+                                          Documents.added_on,
+                                          Documents.content_type)
+                                          .where(Documents.project_id == project_id))
+        all_docs_formatted = []
+        for row in all_documents.all():
+            doc = ProjectDocument(id=row[0],
+                           name=row[1],
+                           added_by=row[2],
+                           added_on=row[3],
+                           content_type=row[4],
+                           project_id=project_id)
+            all_docs_formatted.append(doc)
+        return all_docs_formatted
+    
+
+    def upload_logo(self,
+                    project_id: int,
+                    logo_name: str,
+                    b_content: bytes,
+                    logo_poster:str,
+                    db: Session) -> ProjectLogo:
+        clean_user_provided_name = logo_name.strip().replace(" ","-")
+        logo_key = f"project-{project_id}-logo-{clean_user_provided_name}"
+        q = update(Projects).where(Projects.id == project_id).values(
+                {"logo": logo_key,
+                 "updated_by": logo_poster,
+                 "updated_on": datetime.now()}
+            )
+        try:
+            db.execute(q)
+            upload_file_to_s3("logos-raw", logo_key, b_content)

```

Comment:
These bucket names should be got from Env variables. It is not soo sensitive information, but in production you will use multiple environments (dev, test, prod). And for each environment a new bucket would be used (logos-dev, logos-test, logos-prod). Thus it should not be hardcoded in the code